kuzzleio/kuzzle-admin-console
# Multi-session backend connection

Latest improvements in connection-management led to the unwanted behavior of the app asking to select a connection when opening it in a new browser tab. This PR changes this behavior as follows
* When the app is opened in a new tab, the app connects to the last connected backend (stored in LocalStorage), if this value is missing, the app prompts the user to select a connection among the ones available in the list.
* When a connection is selected, its ID is stored both in the LocalStorage as the last connection, but also in the SessionStorage, as the current connection.
* Upon reload, the apps tries first to connect to the current backend (from SessionStorage): if this value is missing, it falls back to the last connected backend.

### How should this be manually tested?

* Open the app in a new tab (A) and select a connection
* Open the app in another tab (B) and verify it connects to the same backend
* Change the connection of tab B to another backend
* Reload the app in tabs A and B and verify each one connects to its current backend
